pkg/client/clientset/versioned/typed/topology/v1alpha1/fake/fake_hypernode.go

```diff
@@ -18,168 +18,33 @@ limitations under the License.
-	_, err := c.Fake.
-		Invokes(testing.NewRootDeleteActionWithOptions(hypernodesResource, name, opts), &v1alpha1.HyperNode{})
-	return err
-}
-
-// DeleteCollection deletes a collection of objects.
-func (c *FakeHyperNodes) DeleteCollection(ctx context.Context, opts v1.DeleteOptions, listOpts v1.ListOptions) error {
-	action := testing.NewRootDeleteCollectionActionWithOptions(hypernodesResource, opts, listOpts)
-
-	_, err := c.Fake.Invokes(action, &v1alpha1.HyperNodeList{})
-	return err
-}
-
-// Patch applies the patch and returns the patched hyperNode.
-func (c *FakeHyperNodes) Patch(ctx context.Context, name string, pt types.PatchType, data []byte, opts v1.PatchOptions, subresources ...string) (result *v1alpha1.HyperNode, err error) {
-	emptyResult := &v1alpha1.HyperNode{}
-	obj, err := c.Fake.
-		Invokes(testing.NewRootPatchSubresourceActionWithOptions(hypernodesResource, name, pt, data, opts, subresources...), emptyResult)
-	if obj == nil {
-		return emptyResult, err
-	}
-	return obj.(*v1alpha1.HyperNode), err
-}
-
-// Apply takes the given apply declarative configuration, applies it and returns the applied hyperNode.
-func (c *FakeHyperNodes) Apply(ctx context.Context, hyperNode *topologyv1alpha1.HyperNodeApplyConfiguration, opts v1.ApplyOptions) (result *v1alpha1.HyperNode, err error) {
-	if hyperNode == nil {
-		return nil, fmt.Errorf("hyperNode provided to Apply must not be nil")
-	}
-	data, err := json.Marshal(hyperNode)
-	if err != nil {
-		return nil, err
-	}
-	name := hyperNode.Name
-	if name == nil {
-		return nil, fmt.Errorf("hyperNode.Name must be provided to Apply")
-	}
-	emptyResult := &v1alpha1.HyperNode{}
-	obj, err := c.Fake.
-		Invokes(testing.NewRootPatchSubresourceActionWithOptions(hypernodesResource, *name, types.ApplyPatchType, data, opts.ToPatchOptions()), emptyResult)
-	if obj == nil {
-		return emptyResult, err
-	}
-	return obj.(*v1alpha1.HyperNode), err
-}
-
-// ApplyStatus was generated because the type contains a Status member.
-// Add a +genclient:noStatus comment above the type to avoid generating ApplyStatus().
-func (c *FakeHyperNodes) ApplyStatus(ctx context.Context, hyperNode *topologyv1alpha1.HyperNodeApplyConfiguration, opts v1.ApplyOptions) (result *v1alpha1.HyperNode, err error) {
-	if hyperNode == nil {
-		return nil, fmt.Errorf("hyperNode provided to Apply must not be nil")
-	}
-	data, err := json.Marshal(hyperNode)
-	if err != nil {
-		return nil, err
-	}
-	name := hyperNode.Name
-	if name == nil {
-		return nil, fmt.Errorf("hyperNode.Name must be provided to Apply")
-	}
-	emptyResult := &v1alpha1.HyperNode{}
-	obj, err := c.Fake.
-		Invokes(testing.NewRootPatchSubresourceActionWithOptions(hypernodesResource, *name, types.ApplyPatchType, data, opts.ToPatchOptions(), "status"), emptyResult)
-	if obj == nil {
-		return emptyResult, err
+func newFakeHyperNodes(fake *FakeTopologyV1alpha1) typedtopologyv1alpha1.HyperNodeInterface {
```

Comment:
In addition to go.mod/go.sum and the scripts in hack, why are there so many other files changed? Did you change them manually or was it automatically generated?

Reply:
these all were auto generated.

Reply:
OK